doc/source/Network.rst

```diff
@@ -0,0 +1,89 @@
+LHS                                        RHS                                                   Rate   citation                                                                                                                                                                                                                                                                                 Enabled By
+========================================== ===================================================  ======= =====================================================================================================================================================================================================================================================================================  ===========================================================
+H + e\ :sup:`-`                            H\ :sup:`+` + e\ :sup:`-` + e\ :sup:`-`                k1    `Abel et al. (1997) <https://ui.adsabs.harvard.edu/abs/1997NewA....2..181A/abstract>`__                                                                                                                                                                                                :c:data:`primordial_chemistry` >0     
+H\ :sup:`+` +  e\ :sup:`-`                 H +  :math:`{\gamma}`                                  k2    `Abel et al. (1997) <https://ui.adsabs.harvard.edu/abs/1997NewA....2..181A/abstract>`__  & `Hui & Gnedin . (1997) <https://ui.adsabs.harvard.edu/abs/1997MNRAS.292...27H/abstract>`__                                                                                                  :c:data:`primordial_chemistry` >0   
+He +  e\ :sup:`-`                          He\ :sup:`+` +  e\ :sup:`-` +  e\ :sup:`-`             k3    `Abel et al. (1997) <https://ui.adsabs.harvard.edu/abs/1997NewA....2..181A/abstract>`__                                                                                                                                                                                                :c:data:`primordial_chemistry` >0  
+He\ :sup:`+` + e\ :sup:`-`                 He\ :sup:`+` + :math:`{\gamma}`                        k4    `Hui & Gnedin . (1997) <https://ui.adsabs.harvard.edu/abs/1997MNRAS.292...27H/abstract>`_  & `Aldrovandi & Pequignot. (1973) <https://ui.adsabs.harvard.edu/abs/1973A%26A....25..137A/abstract>`_ & `Black. (1981) <https://ui.adsabs.harvard.edu/abs/1981MNRAS.197..553B/abstract>`_  :c:data:`primordial_chemistry` >0   
+He\ :sup:`+` +  e\ :sup:`-`                He\ :sup:`+` :sup:`+` + e\ :sup:`-` + e\ :sup:`-`      k5    `Abel et al. (1997) <https://ui.adsabs.harvard.edu/abs/1997NewA....2..181A/abstract>`__                                                                                                                                                                                                :c:data:`primordial_chemistry` >0     
```

Comment:
```suggestion
He\ :sup:`+` +  e\ :sup:`-`                He\ :sup:`+`\ :sup:`+` + e\ :sup:`-` + e\ :sup:`-`      k5    `Abel et al. (1997) <https://ui.adsabs.harvard.edu/abs/1997NewA....2..181A/abstract>`__                                                                                                                                                                                                :c:data:`primordial_chemistry` >0     
```
I think that inserting this backslash will remove the space between the superscripted + signs in He++